public/js/controllers/charts_controller.js

```diff
@@ -322,6 +327,19 @@ function missedVotesFunc (data) {
   return zipWindowHvY(data.missed, data.window, 1, data.offset * data.window)
 }
 
+function durationBTwBlocksFunc (rawData) {
+  var times = rawData.x
+  var counts = rawData.y
+  var blockCount = counts.reduce((total, n) => total + n)
```

Comment:
`rawData` has `t` and `duration` arrays, but no `x` or `y`.

Reply:
The API hit for this is `http://127.0.0.1:7777/api/chart/duration-btw-blocks?limit=all?bin=day`

Reply:
Looking into this.

Reply:
Still broken in this function.  Are you able to address this, or should we put this PR on hold?